analysis/measures_definition_pf_statistics.py

```diff
@@ -0,0 +1,70 @@
+from ehrql import INTERVAL, create_measures, months
+from ehrql.tables.tpp import (
+    practice_registrations,
+    patients,
+)
+
+from pf_dataset import pharmacy_first_event_codes
+from measures_definition_pf_medications import pharmacy_first_med_codes
+from measures_definition_pf_breakdown import pharmacy_first_conditions_codes, selected_events
+
+measures = create_measures()
+measures.configure_dummy_data(population_size=1000)
+
+start_date = "2023-11-01"
+monthly_intervals = 12
+
+registration = practice_registrations.for_patient_on(INTERVAL.end_date)
+
+# Loop through all codes in each sublist of the dictionary to flatten the list ready for is_in commands to be used and have a list of pf_condition codes
+pf_condition_codelist = [code for sublist in pharmacy_first_conditions_codes.values() for code in sublist]
+
+# Create variable which contains boolean values of whether pharmacy first event exists for patient
+has_pf_consultation = selected_events.where(
+    selected_events.snomedct_code.is_in(
+        pharmacy_first_event_codes["combined_pf_service"]
+    )
+).exists_for_patient()
+
+# PF consultations with PF clinical condition
+has_pf_condition = selected_events.where(
+    selected_events.snomedct_code.is_in(
+        pf_condition_codelist
+    )
+).exists_for_patient()
+
+# PF consultations with prescribed PF medication
+has_pf_medication = selected_events.where(
+    selected_events.snomedct_code.is_in(
+        pharmacy_first_med_codes
+    )
+).exists_for_patient()
+
+# PF consultations with PF clinical condition and PF medication
+has_pf_condition_and_medication = has_pf_condition & has_pf_medication
```

Comment:
because we're only using this once we could just combine this directly into the numberator of the measures below?